pages/store/[nftClassId]/index.vue

```diff
@@ -174,16 +174,40 @@
                           class="text-sm font-semibold"
                           v-text="$t('product_page_sold_out_button_label')"
                         />
-                        <span v-else>
-                          <span
-                            class="text-xs mr-0.5"
-                            v-text="item.currency"
-                          />
-                          <span
-                            class="font-semibold"
-                            v-text="item.formattedPrice"
-                          />
-                        </span>
+                        <div
+                          v-else
+                          class="flex flex-col items-end text-right"
+                        >
+                          <div v-if="isLikerPlus && item.discountedPrice">
```

Comment:
Not sure if `<template>` is enough, and also we might want to split `item.discountedPrice` and `isLikerPlus` check into ` v-text="item.discountedPrice"` and `PlusBadgeIcon`